eng/WpfArcadeSdk/tools/Packaging.targets

```
@@ -320,28 +327,28 @@ $(PreparePackageAssetsDependsOn):
         <PackagePath>$(LibFolder)</PackagePath>
       </Content>
 
-      <Content Include="$(ArtifactsPackagingDir)$(NormalizedPackageName)\runtimes\**\*.dll"
-         Condition="('$(CreateArchNeutralPackage)'!='true' Or '$(IncludeAssembliesInArchNeutralPackage)'== 'true') and Exists('$(ArtifactsPackagingDir)$(NormalizedPackageName)\runtimes\')" >
+      <Content Include="$(ArtifactsPackagingDir)$(NormalizedPackageName)\$(DestinationRuntimeSubFolder)\*.dll"
```

Should we use `$(DestinationRuntimeSubFolder)\**` ?

```suggestion
      <Content Include="$(ArtifactsPackagingDir)$(NormalizedPackageName)\$(DestinationRuntimeSubFolder)\**\*.dll"
```